resources/metric.mjs

```diff
@@ -1,7 +1,7 @@
 import * as Statistics from "./statistics.mjs";
 
 /** Number of milliseconds in a single minute. */
```

Comment:
I don't think we need the comment now that the variable is appropriately named.

Reply:
removed.